imager/imager/templates/album.html

```diff
@@ -0,0 +1,23 @@
+{% extends "home.html" %}
+{% block content %}
+<h1>Hello {{user.username}}</h1>
+
+<ul>
+  <li><b>Image Count:</b>  {{img_qty}}</li>
+</ul>
+
+<section>
+  <h2>Photos in this Album</h2>
+  <ul>
+    {% for photo in photos %}
+      <li><a href="/images/photo/{{photo.id}}"><img src="{{photo.image.url}}" alt="" width="100px" height="100px" /></a></li>
+    {% endfor %}
+  </ul>
+  <a href="/images/albums/edit/{{album.id}}">Edit Album</a>
+  <form action="/images/albums/delete/{{album.id}}" method="post">
+    {% csrf_token %}
+    <input type="submit" value="KILL IT WITH FIRE!">
+  </form>
+</section>
```

Comment:
This is probably a good template in which to display album information (title, description, etc.).